src/tools/sfdk/sfdkconstants.h

```diff
@@ -35,6 +35,8 @@ const char GENERAL_DOMAIN_NAME[] = "general";
 const char BUILD_ENGINE_SYSTEM_BUS_CONNECTION[] = "build-engine-system-bus";
 const char SAILFISH_SDK_SFDK_DBUS_SERVICE[] = "SAILFISH_SDK_SFDK_DBUS_SERVICE";
 
+const char WORKSPACE_PSEUDO_VARIABLE[] = "$WORKSPACE";
```

Comment:
I'm not sure if using $ in the pseudo variable name is a good idea. It requires escaping if used from shell, and it's easy to confuse with environment variables. On the other hand, you probably had good reasons for choosing this approach - I would just like to hear them :)

Reply:
Yup, I considered `%{WORKSPACE}`, which is the syntax used by qtc's macro expander, `@WORKSPACE@`, which is used by Qt IFW, but I avoided that exactly for the possibility for confusion and need to escape in installer code. I chose sh-like syntax as the same approach is used for `$ORIGIN` in RPAHT.